.github/workflows/sdk.yaml

```diff
@@ -43,8 +43,19 @@ jobs:
       - name: Display rustc version
         run: rustc --version
       - name: Test gax with features disabled
-        run: cargo test --package google-cloud-gax
-      - run: cargo test --workspace
+        run: cargo clean && cargo test --package google-cloud-gax
+      - name: Test gax-internal with no features
+        run: cargo clean && cargo test --package google-cloud-gax-internal --no-default-features
```

Comment:
As of now, there are no default features, so these cases are equivalent.

Consider removing the `--no-default-features` check until we add a default feature. (Which is hopefully never).

Reply:
D'oh, I see now that one package is `google-cloud-gax` and the other is `google-cloud-gax-internal`. Ignore me.